### ISSUE TITLE
Keyboard shortcut for Zoom In (⌘+) isn't working on Mac

### ISSUE THREAD

**Issue by lol-russo** [Zettlr/Zettlr#715]
## Description
The keyboard shortcut for zooming in isn't working for me. Zoom Out (⌘-) and Reset Zoom (⌘0) both work perfectly, but Zoom In just doesn't work. However, when you select `View -> Zoom in` from the menu bar it works fine.

## Reproducing
1. Open any file
2. Press `Command` and `+` keys (⌘+). Nothing happens
3. Press  `Command` and `-` keys (⌘-). View zooms out as expected
4. Press `Command` and `+` keys (⌘+). Nothing happens again (I thought that perhaps you couldn't zoom further in than the default, and this is what causes it - but no, it doesn't work here either)
5. Press  `Command` and `0` keys (⌘0). View returns to default zoom level as expected

## Expected behaviour
When a user presses the shortcut for zooming in (⌘+) the view should zoom in a set percentage (usually by 10%)

**Screenshots**

## Platform
 - OS and version: macOS 10.15.3
 - Zettlr Version:  1.6.0
 - Screen Resolution: 5120 x 2880 (iMac)

## Additional information
Thank you for an awesome application! I'm enjoying it so far and love that it is OSS ♥️

**Comment by danieltomasz** [comment]
This is still the case with the latest beta.

**Comment by danieltomasz** [comment]
Quick solution:

>Correct the current item to reflect that it needs to be triggered with ⇪⌘+; current trigger key combo remains unchanged.

This is the electron issue https://github.com/electron/electron/issues/15496

**Comment by danieltomasz** [comment]
@lol-russo  it actually working, due how mac keyboard work  we actually typing  ⌘= on mac, to get "real" +, there is needed combination "⇪⌘+"

**Comment by nathanlesage** [comment]
@danieltomasz Ah, I see. If that's the reason, then I think we can close this issue, because this is (in my humble opinion) not a valid reason to try something (e.g. on a German keyboard layout, opening up the sidebar requires the combination `⇪⌘ß`, and not just `⌘?` as on the English keyboard, so I think this is somehow to be expected!

**Comment by benwhalley** [comment]
As feedback, I came here to report this as a bug. I think the conventional Mac shortcut would be ⌘+  but not requiring the shift even when the + character would require that on an English keyboard. See Safari's behaviour for example, or the Mail.app shortcut which are probably the two most commonly used apps on os x.

That said, Apple breaks it's own UI guidelines to do this, which say:

> Identify a two-character key by its lower character unless Shift is part of the shortcut. For example, the keyboard shortcut for Hide Status Bar is Command-Slash (/). If the Shift key is part of the keyboard shortcut, identify the key using the upper of the two characters. For example, the keyboard shortcut for Help is Command-Question mark (?), not Shift-Command-Slash.

from https://developer.apple.com/design/human-interface-guidelines/macos/user-interaction/keyboard/

This means that increase Zoom in safari should really be labelled `⌘=`

FWIW I think either changing the Mac shortcut to work `⌘+` or `⌘=` would be best ... I've had a Mac for a long time and I didn't expect the current behaviour and didn't expect `⌘+` to mean I had to also press the shift key.
